site/docs/4.3/components/dropdowns.md

```diff
@@ -662,7 +662,7 @@ Add a header to label sections of actions in any dropdown menu.
 
 {% capture example %}
 <div class="dropdown-menu">
-  <h6 class="dropdown-header">Dropdown header</h6>
+  <h4 class="dropdown-header">Dropdown header</h4>
```

Comment:
i seem to remember a recent discussion about having these as `<h6>` to avoid having them show up in the TOC, but can't seem to find it just now.

these headings are, weirdly, not really part of the document structure per se, but in an example...but as HTML has no way of defining this difference (I guess if it were an iframe, perhaps it would be more logical in a way if the heading there in the separate example has, say, `<h1>` or something...). As it's not a huge problem (nobody dies) that this is `<h6>` rather than `<h4>`, I think we can live with the way it currently is (to avoid it showing in the TOC)

Reply:
I agree nobody will die :-)
There's a config in _config.yml that prevent example heading to be displayed in toc
```
toc:
  min_level:        2
  max_level:        4
  no_toc_section_class:
    - "bd-callout"
    - "bd-example"
```

Reply:
Please revert this too.